### PR TITLE
[CARBONDATA-1478] Update compaction documentation

### PR DESCRIPTION
**Pull request by xubo245**
In order to easier to understand, update the description about compaction.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/107/

**Comment by ravipesala** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder1/8/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/734/

File: docs/data-management.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ CLEAN FILES FOR TABLE table1
    
 | Parameter | Default | Application | Description | Valid Values |
 |-----------------------------------------|---------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
-| carbon.compaction.level.threshold | 4, 3 | Minor | This property is for minor compaction which decides how many segments to be merged. Example: If it is set as 2, 3 then minor compaction will be triggered for every 2 segments. 3 is the number of level 1 compacted segment which is further compacted to new segment. | NA |
+| carbon.compaction.level.threshold | 4, 3 | Minor | This property is for minor compaction which decides how many segments to be merged. Example: If it is set as 2, 3, then minor compaction will be triggered for every 2 segments in level 1. 3 is the number of level 1 compacted segment which is further compacted to new segment in level 2. | NA |
```
Comment (jackylk) [inline docs/data-management.md]: suggest to change `2, 3` to `"2, 3"`

Comment (jackylk) [reply docs/data-management.md]: suggest to explain level 1 compaction and level 2 compaction first.

Comment (xubo245) [reply docs/data-management.md]: I just have changed 2, 3 to "2, 3".

" Minor Compaction" part has explained  level 1 compaction and  and level 2 compaction

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/156/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/786/

**Pull request closed without merging**